ReactWindows/ReactNative.Shared/Views/Text/ReactSpanViewManager.cs

```diff
@@ -1,12 +1,12 @@
-﻿using ReactNative.Reflection;
+using ReactNative.Reflection;
 using ReactNative.UIManager;
 using ReactNative.UIManager.Annotations;
 using System;
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
-
 #if WINDOWS_UWP
+using ReactNative.Common;
 using Windows.UI.Xaml;
```

Comment:
We should be consistent whether we're including the ReactNative.Common interface, or partially qualifying the calls to AccessibilityHelper. I think this PR has a mixed approach. My preference is to include the namespace as is done here.

Reply:
Fixed

---
In reply to: [170250680](https://github.com/Microsoft/react-native-windows/pull/1645#discussion_r170250680) [](ancestors = 170250680)